File: how_many_words/solutions/how_many_words.cs
using System;
class MainClass {
  static void Main () {
    String word = "The dog jumps over the cat.";
    
    int the_first_answer = count_words(word);
    int the_second_answer = count_word(word);
    Console.WriteLine(the_first_answer);
    Console.WriteLine(the_second_answer);
  }
  
  public static int count_words(string phrase){
    string[] words = phrase.Split(' ');
    return words.Length;
  }
  
  public static int count_word(string phrase){
    int num_of_words = 1;
    foreach (char letter in phrase){
      if(letter == ' ') {
        num_of_words = num_of_words + 1;
      }
    }
    return num_of_words;
  }
  
}

File: how_many_words/solutions/how_many_words.js
var word = "The dog jumps over the cat.";

var the_first_answer = words_count(word);
var the_second_answer = word_count(word);

console.log(the_first_answer);
console.log(the_second_answer);

function words_count(word){
  words = word.split(' ');
  return words.length;
}

function word_count(word){
  num_of_words = 1;
  for(var i = 0; i < word.length; i++){
    if(word[i] === ' '){
      num_of_words += 1;
    }
  }
  return num_of_words;
}